packages/@aws-cdk/aws-appmesh/lib/mesh.ts

```diff
@@ -40,6 +40,13 @@ export interface IMesh extends cdk.IResource {
    */
   readonly meshArn: string;
 
+  /**
+   * The AWS IAM account ID of the AppMesh mesh
```

Comment:
Small nit: We could include an `@see https://docs.aws.amazon.com/app-mesh/latest/userguide/sharing.html` here. I don't know if we do that in many other places, but could help customers

Reply:
Well noted. I will add the link.